### DISCUSSION TITLE
Mark installations as 'Always keep'

### DISCUSSION THREAD

**Discussion by W1tchseason** [Klocman/Bulk-Crap-Uninstaller#646] [Ideas]
First, thanks for a great app. Works really well & GUI is intuitive - 'wizard' also useful.
Suggestion - I have quite a lot of stuff installed on my PC and there are several things that I will always want to keep; it would be useful to be able to mark these in some way and, if desired, hide them from the list.
Also, some of the things in my list don't have a meaningful name - several just show the relevant registry entry. There is a 'rename' option but this doesn't seem to work - I get an error message telling me to select only one item in the list when I *have* only selected one item. So I don't know if this should do what It'd like it to - e.g. to be able to rename these garbage items in the list.
Thanks again :-)

**Comment by Klocman** [comment]
At the moment you can achieve this by using advanced filters and excluding these applications. There would need to be a separate system for simple filtering mode for this, maybe someone could make a PR to add it.

**Comment by Klocman** [comment]
Moving discussion to an issue #935
